src/main/webapp/resources/css/final_bill_custom4_letterhead.css

```
@@ -0,0 +1,163 @@
+           margin box does. The composite's own inline margin-bottom: 2.5cm is
+           separate again: it's trailing space after the very last page's
+           content ends, not a per-page reservation. */
+    }
+
+    /* Keep a single line item / doctor-fee row from being split across a page
+       boundary when a long bill overflows onto a second sheet */
+    [id$="originalCustom4BillPriview"] tr,
+    [id$="duplicateCustom4BillPriview"] tr,
+    [id$="custom4BillOnly"] tr,
+    [id$="custom4BillOnlyDuplicate"] tr {
+        page-break-inside: avoid;
```

Stylelint (17.14.1)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)